index.js

```diff
@@ -78,6 +78,20 @@ exports.socketio = function (hook_name, args, cb){
       });
     });
 
+    socket.on('commentTextUpdated', function(data, callback) {
```

Comment:
Following the standard of other events, this one should be `updateCommentText`, not `commentTextUpdated`